### PR TITLE
Fix where pgbouncer URLs were not available cross-namespace

### PR DESCRIPTION
**Pull request by dimberman**
**Checklist**

- [x]  Chart.yaml version updated

### CODE REVIEW

File: templates/secrets/metadata-connection-secret.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
 {{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
+
```
Comment (andriisoldatenko) [inline templates/secrets/metadata-connection-secret.yaml]: what do you think to simplify code using extract to variables?

Comment (andriisoldatenko) [reply templates/secrets/metadata-connection-secret.yaml]: {{- $host := (ternary $pgbouncer_host $host .Values.pgbouncer.enabled) }}

Comment (andriisoldatenko) [reply templates/secrets/metadata-connection-secret.yaml]: {{- $port := (((ternary .Values.ports.pgbouncer .Values.data.metadataConnection.port .Values.pgbouncer.enabled) | toString) }}

Comment (andriisoldatenko) [reply templates/secrets/metadata-connection-secret.yaml]: {{- $db := (ternary (printf "%s-%s" .Release.Name "metadata") .Values.data.metadataConnection.db .Values.pgbouncer.enabled) }}

Comment (andriisoldatenko) [reply templates/secrets/metadata-connection-secret.yaml]: wdyt @schnie ?

Comment (ashb) [reply templates/secrets/metadata-connection-secret.yaml]: Yeah, given the _monster_ that is the `connection` attribute, not having that needing all the `ternary` there would do wonders for the readability of that line.

File: templates/secrets/metadata-connection-secret.yaml [hunk 2]
```diff
@@ -2,7 +2,12 @@
 ## Airflow Metadata Secret
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
-{{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $postgres_host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
+{{- $host := ternary $pgbouncer_host $postgres_host .Values.pgbouncer.enabled }}
+{{- $port := ((ternary .Values.ports.pgbouncer .Values.data.metadataConnection.port .Values.pgbouncer.enabled) | toString) }}
+{{- $database := (ternary (printf "%s-%s" .Release.Name "metadata") .Values.data.metadataConnection.db .Values.pgbouncer.enabled) }}
```
Comment (ashb) [inline templates/secrets/metadata-connection-secret.yaml]: Does this need to change for pgbouncer support? I thought the DB name is the same regardless?

Comment (ashb) [reply templates/secrets/metadata-connection-secret.yaml]: Guess this isn't a change thought

Comment (dimberman) [reply templates/secrets/metadata-connection-secret.yaml]: @ashb that part doesn't NEED to change but frankly, the gnarliness of that line is what allowed this bug to exist in the first place. Figured it was good to do some housekeeping while I'm changing the file anyways.

File: templates/secrets/metadata-connection-secret.yaml [hunk 3]
```diff
@@ -2,7 +2,12 @@
 ## Airflow Metadata Secret
 #################################
 {{- if (and .Values.data.metadataConnection (not .Values.data.metadataSecretName)) }}
-{{- $host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $postgres_host := .Values.data.metadataConnection.host | default (printf "%s-%s.%s.svc.cluster.local" .Release.Name "postgresql" .Release.Namespace) }}
+{{- $pgbouncer_host := (printf "%s-%s.%s.svc.cluster.local" .Release.Name "pgbouncer" .Release.Namespace) }}
```
Comment (schnie) [inline templates/secrets/metadata-connection-secret.yaml]: @dimberman small nit, but I think most values and variables are camelCase in the chart now.

Comment (dimberman) [reply templates/secrets/metadata-connection-secret.yaml]: @schnie fixed :)

**Pull request merged**